src/stories/CardSimple.stories.js

```diff
@@ -0,0 +1,83 @@
+import UnnnicSimpleCard from '../components/Card/SimpleCard.vue';
+import UnnnicButton from '../components/Button/Button.vue';
+
+export default {
+  title: 'Data Display/SimpleCard',
+  component: UnnnicSimpleCard,
+  tags: ['autodocs'],
+  argTypes: {
+    title: { control: { type: 'text' } },
+    titleTooltip: { control: { type: 'text' } },
+    text: { control: { type: 'text' } },
+    titleSize: { control: { type: 'text' } },
+    clickable: { control: { type: 'boolean' } },
+  },
+  render: (args) => ({
+    components: {
+      UnnnicSimpleCard,
+    },
+    setup() {
+      return { args };
+    },
+    template: `
+        <UnnnicSimpleCard v-bind="args">
+        </UnnnicSimpleCard>
+    `,
+  }),
```

Comment:
A simple render like this does not need to be explicit, the storybook already does exactly the same thing if the render is not declared.